build-npm-module/index175.js

```diff
@@ -8,7 +8,6 @@ const getSentenceCase = (str) => {
   return lowerStr.charAt(0).toUpperCase() + lowerStr.slice(1)
 };
 
-const properCaseArr = (str) => str.split(' ').map((word) => getSentenceCase(word));
-
-// Create getProperCase() function to join properCaseArr into spce separated sentence.
+const getProperCase = (str) => {};
 
+// First we can split the input string by space, and call our `getSentenceCase` on each word, store the result array in a local variable called `properCaseArr`.
```

Comment:
This one might also benefit from being broken down into two steps. Still, here's the current text:

Split the input `str` into an array of words, call `getSetenceCase` on each word, and store the resulting array as a local variable called `properCaseArr`.